### ISSUE TITLE
run Stage II: FD-GAN pretraining

### ISSUE THREAD

**Issue by qiangqiangzhu** [yxgeee/FD-GAN#9]
I run "python train.py --display-port 6006 --display-id 1 \
	--stage 1 -d market1501 --name /directory/name/of/saving/checkpoints/ \
	--pose-aug gauss -b 256 -j 4 --niter 50 --niter-decay 50 --lr 0.001 --save-step 10 \
	--lambda-recon 100.0 --lambda-veri 0.0 --lambda-sp 10.0 --smooth-label \
	--netE-pretrain /path/of/model_best.pth.tar" for train,
 and I use your dataset and model.
I just change batchsize from 256 to 2.
There are some errors, I don't know how to solve them.

![screenshot from 2018-11-11 16-05-45](https://user-images.githubusercontent.com/43439219/48310727-a63f7380-e5ce-11e8-9c08-dce4fb46239e.png)

Thanks.

**Comment by yxgeee** [comment]
It seems to be caused by `visdom`.

Before running the code, you need to start the visdom by openning a new window and type
```
python -m visdom.server -port=6006
```
where `-port` should be consistent with `--display-port` (details refer to https://github.com/facebookresearch/visdom)

Also you can disable the visdom by `--display-id 0`

**Comment by qiangqiangzhu** [comment]
> It seems to be caused by `visdom`.
> 
> Before running the code, you need to start the visdom by openning a new window and type
> 
> ```
> python -m visdom.server -port=6006
> ```
> 
> where `-port` should be consistent with `--display-port` (details refer to https://github.com/facebookresearch/visdom)
> 
> Also you can disable the visdom by `--display-id 0`

thank you ,I have already solved the problem.But when I run this stage has produced a new problem, pictures below.
![screenshot from 2018-11-14 12-11-37](https://user-images.githubusercontent.com/43439219/48459752-7f6b8200-e806-11e8-8348-dfc3de0c3a51.png)

and Print this l1 function of input
Variable containing:[torch.cuda.FloatTensor with no dimension

![screenshot from 2018-11-14 12-11-25](https://user-images.githubusercontent.com/43439219/48459807-c8bbd180-e806-11e8-9ab1-f525a5d19b28.png)

**Comment by yxgeee** [comment]
Your batch size is too small, as a result that `fake_1 = self.fake[:self.fake.size(0)//2]` gets empty as well as `fake_2`.
